packages/opentelemetry-exporter-collector/test/browser/CollectorMetricExporter.test.ts

```diff
@@ -93,10 +93,11 @@ describe('CollectorMetricExporter - web', () => {
       it('should successfully send metrics using sendBeacon', done => {
         collectorExporter.export(metrics, () => {});
 
-        setTimeout(() => {
+        setTimeout(async () => {
           const args = stubBeacon.args[0];
           const url = args[0];
-          const body = args[1];
+          const blob:Blob = args[1];
```

Comment:
Is this missing a space?

Reply:
If it passes lint it's generally fine but it can't hurt to do I suppose

```suggestion
          const blob: Blob = args[1];
```